ibis/expr/format.py

```diff
@@ -559,6 +559,13 @@ def _fmt_value_sort_key(op: ops.SortKey, *, aliases: Aliases) -> str:
     return f"{sort_direction}|{expr}"
 
 
+@fmt_value.register
+def _fmt_topk(op: ops.TopK, *, aliases: Aliases) -> str:
+    arg = fmt_value(op.arg, aliases=aliases)
+    by = fmt_value(op.by, aliases=aliases)
+    return f"TopK({arg}, {op.k}, {by})"
```

Comment:
@cpcloud just quickly added an implementation since I changed the base class of TopK from ValueOp to Node.